lib/service_sign_in/check-update-company-car-tax.cy.yaml

```diff
@@ -0,0 +1,51 @@
+start_page_slug: diweddaru-manylion-car-cwmni
+locale: cy
+choose_sign_in:
+  title: Profi pwy ydych er mwyn mynd yn eich blaen
+  slug: profi-pwy-ydych
+  options:
+    - text: Defnyddio Porth y Llywodraeth
+      url: https://www.tax.service.gov.uk/account
+      hint_text: Bydd gennych Ddynodydd Defnyddiwr (ID) os ydych wedi cofrestru i wneud pethau fel cyflwyno'ch Ffurflen Dreth Hunanasesiad ar-lein.
+    - text: Defnyddio GOV.UK Verify
+      url: https://www.tax.service.gov.uk/ida/sa/login
```

Comment:
Same thing as above, I would go with `https://www.tax.service.gov.uk/paye/company-car/start-verify` to stay consistent and make sure if the internal address changes, HMRC can still handle/redirect our traffic